README.md

```diff
@@ -1,4 +1,34 @@
-# iroha2-block-explorer-backend
+# Iroha 2 Blockchain Explorer Backend
+
+This readme provides an overview of the Iroha 2 blockchain explorer backend and instructions on how to install, run, and utilize the explorer's backend features.
+
+## Installation
+
+To set up the Iroha 2 blockchain explorer backend, follow these steps:
+
+1. Install Rust.
+2. For full functionality of the explorer backend, build [Iroha_RC.9](https://github.com/hyperledger/iroha/tree/ea45b5053018acd48340024800786ff5a3d0904d) and ensure it is running.
```

Comment:
```suggestion
2. For full functionality of the explorer backend, build [Iroha `v2.0.0-pre-rc.9`](https://github.com/hyperledger/iroha/tree/ea45b5053018acd48340024800786ff5a3d0904d) and ensure it is running.
```

You can check how the version is actually written in hyperledger/iroha `Cargo.toml`s.